mzLib/MassSpectrometry/PeakIndexing/ExtractedIonChromatogram.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MathNet.Numerics.Interpolation;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MassSpectrometry
+{
+    public class ExtractedIonChromatogram
+    {
+        public List<IIndexedPeak> Peaks { get; set; }
+
+        public double ApexRT => Peaks.OrderByDescending(p => p.Intensity).First().RetentionTime;
+        public int ApexCycle => Peaks.OrderByDescending(p => p.Intensity).First().ZeroBasedScanIndex;
```

Comment:
I don't know what ApexCycle means. I think ApexScanIndex would be more descriptive